src/wamr/wamr.cc

```diff
@@ -56,11 +56,11 @@ class Wamr : public WasmVm {
 public:
   Wamr() {}
 
-  std::string_view runtime() override { return "wamr"; }
+  std::string_view runtime() override { return "envoy.wasm.runtime.wamr"; }
```

Comment:
Why changing to `envoy.wasm.runtime.wamr`, it seems to be different from wasmtime.cc or v8.cc implementation.